index.js

```diff
@@ -67,8 +67,8 @@ function medianof3(a, lo, mid, hi) {
 /*
 * Heapsort algorithm
 */
-function heapsort(a, lo, hi) {
-    var n = hi-lo, i;
+const heapsort = (a, lo, hi) => {
+    const n = hi-lo, i;
```

Comment:
Update this as you cannot initialize with **const** and not assign a value to it. It will throw an error by the way.